src/main/java/uk/gov/ch/model/officer/active/ActiveOfficerDetails.java

```diff
@@ -252,9 +328,14 @@ public Address getResidentialAddress() {
     }
 
     private String formatDate(String dateString) throws ParseException {
-        SimpleDateFormat formatFrom = new SimpleDateFormat("yyyy-MM-dd");
-        Date date = formatFrom.parse(dateString);
-        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("dd MMMMM yyyy");
-        return simpleDateFormat.format(date);
+        if (dateString != null) {
```

Comment:
you can use a guard clause here, if datestring == null, return null immediately, then do the date check without needing the else {}